### ISSUE TITLE
How to uninstall tpm bypass cmd script - this has hosed my system and cant install windows now

### ISSUE THREAD

**Issue by davidm71** [AveYo/MediaCreationTool.bat#301]
Hi,
The TPM bypass script has hosed my ability to run systemprep or setuo. It just fails after being blocked. 

Thank you.

**Comment by AveYo** [comment]
Just run the script again, it has built-in undo, it should be known.

To manually remove it, in an admin CMD / PowerShell enter:
`reg delete "HKLM\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Image File Execution Options\SetupHost.exe" /f`

Would you mind detailing the "fails after being blocked"?  
Is your AV blocking it?
What version are you trying to setup?

**Comment by davidm71** [comment]
Well I was trying to do an in place upgrade from Win 10 to 11 using the setupprep application in the sources subfolder of version 22631 that I acquired from the UUP download site and optimized in NtLite.  Was having difficulty getting the built in tpm checks to be dismissed. Tried deleting appraiserres.dll without luck. So read that your script can help with overcoming that restriction. Unfortunately after I ran the cmd file when I tried to run the Windows in place setup program a small black cmd rectangle would pop up and disappear and the setup program not even run like it was being blocked. Didn't have AV installed and before running that script no issues getting setup to run. Only thing that worked was running the full Windows installer, not the setupprep, and had Windows installer basically do a fresh install where it copies all the old windows files into Windows.old not retaining any past installed programs. I only hope all traces of that script are now gone.
